modules/ROOT/partials/aws-s3-credentials.adoc

```diff
@@ -0,0 +1,39 @@
+. Ensure that the AWS CLI is installed and configured with the necessary credentials.
+.. Install the AWS CLI by following the instructions in the AWS official documentation -- link:https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html[Install the AWS CLI version 2].
+.. Use `aws configure` command to set your `aws_access_key_id` and `aws_secret_access_key` from AWS.
+.. Create an S3 bucket and a directory to store the backup files using the AWS CLI:
++
+[source,shell]
+----
+aws s3 mb --region=us-east-1 s3://myBucket
+aws s3api put-object --bucket myBucket mat --key myDirectory/
```

Comment:
Sorry - meant that it should be:
`aws s3api put-object --bucket myBucket --key myDirectory/`

i.e. no mat